includes/class-wc-siftscience-options.php

```diff
@@ -26,7 +26,8 @@ class WC_SiftScience_Options {
 		public static $guid = 'siftsci_guid';
 		public static $stats = 'siftsci_stats';
 		public static $stats_last_sent = 'siftsci_stats_last_sent';
-
+		//min order value input id Rjamleh
+		public static $min_order_value = 'min_order_value';
```

Comment:
This value is used in the options database table. It should start with 'siftsci_' like all the others so it doesn't conflict with other plugins.